style.css

```diff
@@ -87,3 +88,212 @@ body {
   align-items: baseline;
   justify-content: flex-start;
 }
+
+.projects-section {
+  display: flex;
+  flex-direction: column;
+  padding: 16px;
+  gap: 12px;
+  margin-top: 650px;
+  width: 327px;
+  height: 552px;
+  background: #fff;
+  border: 1px solid #dfe1e6;
+  border-radius: 16px;
+}
+
+.cards {
+  width: 295px;
+  height: 220px;
+  border-radius: 8px;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+}
+
+.cards-details {
+  display: flex;
+  flex-direction: column;
+  gap: 12px;
+  width: 295px;
+  flex: none;
+  order: 1;
+  flex-grow: 0;
+}
+
+.title {
+  width: 295px;
+  height: 36px;
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 32px;
+  line-height: 44px;
+  color: #172b4d;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+}
+
+.highlights {
+  display: flex;
+  flex-direction: row;
+  gap: 12px;
+  width: 236px;
+  flex: none;
+  order: 1;
+  flex-grow: 0;
+}
+
+.highlight-text {
+  width: 55px;
+  height: 16px;
+  text-transform: uppercase;
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 16px;
+  color: #344563;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+}
+
+.counter {
+  width: 8px;
+  height: 8px;
+  background: #c1c7d0;
+  flex: none;
+  flex-grow: 0;
+}
+
+.countertext {
+  width: 88px;
+  height: 16px;
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 16px;
+  color: #7a869a;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+}
+
+.cards-descriptions {
+  width: 295px;
+  height: 76px;
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 15px;
+  line-height: 24px;
+  color: #344563;
+  flex: none;
+  order: 1;
+  flex-grow: 0;
+}
+
+.grid-skills {
+  display: grid;
+  grid-template-columns: auto auto auto;
+  grid-gap: 10px;
+  padding: 10px;
+  align-items: center;
+  list-style: none;
+  margin: -20px 0 0 -10px;
+  gap: 8px;
+}
+
+.grid-skills > div {
+  text-align: center;
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-size: 12px;
+  padding: 4px 12px;
+  width: 45px;
+  height: 24px;
+  font-weight: 500;
+  letter-spacing: 0.03rem;
+  line-height: 16px;
+  color: #6070ff;
+  background: #ebebff;
+  border-radius: 8px;
+}
+
+#popup {
+  box-sizing: border-box;
+  border: 1px solid #7f8cff;
+  background-color: #fff;
+  letter-spacing: 0.03rem;
+  line-height: 24px;
+  font-family: "Poppins", sans-serif;
+  border-radius: 8px;
+  padding: 12px;
+  bottom: 10px;
+  gap: 10px;
+  cursor: pointer;
+  text-align: center;
+  font-size: 17px;
+  font-weight: 500;
+  font-style: normal;
+  color: #396df2;
+}
```

Comment:
This is a nice button, I love it 😄 but it could look/behave even better and offer a better UX/UI. Yours looks too large for now. See the differences 👇🏽 

## Figma 
![Screenshot 2023-02-17 at 20 50 12](https://user-images.githubusercontent.com/45739963/219825349-d5258193-9de9-4a2d-9c48-3e917d1770fb.png)

## Your design
![Screenshot 2023-02-17 at 21 46 04](https://user-images.githubusercontent.com/45739963/219827746-281e6bbf-d1a3-4aa0-9219-9c8a83e98098.png)

Also, make sure the button (also the `Get resume` button) follows the button styling: Active, Hover, and Pressed (see below 👇🏽 )

![Screenshot 2023-02-17 at 20 52 56](https://user-images.githubusercontent.com/45739963/219825480-90aea75f-5322-4e74-b6c2-5a8ebc18523b.png)

You can accomplish that by adding the `#popup:hover` style 💯